vmdb/spec/support/api_spec_helper.rb

```diff
@@ -19,7 +19,7 @@ module ApiSpecHelper
 
   def parse_response
     @code    = last_response.status
-    @result  = JSON.parse(last_response.body)
+    @result  = (@code != 204) ? JSON.parse(last_response.body) : {}
     @success = @code < 400
```

Comment:
I thought we had constants available for these codes...would make the code easier to understand at least.